components/automate-ui/src/app/modules/user/user-table/user-table.component.html

```diff
@@ -1,11 +1,11 @@
-<app-authorized [allOf]="getPermissionsPath" [overrideVisible]="overridePermissionsCheck">
+<app-authorized [allOf]="getPermissionsPath">
   <ng-container *ngIf="showEmptyMessage">
     <div class="empty-state">
       <p>Add some users to get started!</p>
```

Comment:
I noticed I still see this message when I'm logged in as the `test` user and I navigate to a team without any users
<img width="1058" alt="Screen Shot 2020-05-20 at 4 52 35 PM" src="https://user-images.githubusercontent.com/21015366/82508775-f13b2080-9aba-11ea-94e9-604d45c1b979.png">

Reply:
nice catch, we should leverage the similar msg we display on projects. ill add that here later tonight.

Reply:
ℹ️ I'm looking into this now (Tuesday morning)

Reply:
❓ I'm not sure I see what "similar msg" you've been referring to, @susanev.

When browsing a project's details as the test user, I find this, no message at all:
![image](https://user-images.githubusercontent.com/870638/82875680-fe7f6180-9f37-11ea-874f-f6861eda36ae.png)

Reply:
💡 You've meant this one:

![image](https://user-images.githubusercontent.com/870638/82876038-72216e80-9f38-11ea-80cf-74ccead39196.png)

I'll draft a similar message here that we can discuss later :)

Reply:
![image](https://user-images.githubusercontent.com/870638/82876370-f4aa2e00-9f38-11ea-8439-3e077a5a130a.png)

No, I believe this will also be shown on the users list view if there's **no local users at all**. However, I believe we can ignore that case -- there's supposed to always be a local admin at least. 🤔 I still don't like this. @msorens what do you think? How would we got about refactoring this properly?

Reply:
My preference would be to fix this since it violates [OCP](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle): this class should not know anything about its consumers.  So I pushed up [a fix](https://github.com/chef/automate/pull/3736/commits/db68dff2e00b5e9894f499ee3033d42446dae85f) that let's the two consumers provide different messages.

➡️ @susanev Please review the text of both messages in that commit.

Now, that said, separately there still might be some display logic to adjust. Here is what I found what I visited each resource page when logged in as a user having no permissions then navigating directly to each page:
<img width="725" alt="image" src="https://user-images.githubusercontent.com/6817500/82957444-d27bd480-9f67-11ea-848c-2a6fb8b900a4.png">
Clearly there is some work to be done there on four of those pages, but for the purposes of _this_ PR we might want to address why the Users list is blank and is not showing the message (with or without my commit).

Reply:
the messages look good to me, thank you everyone for the assists!!

Reply:
@msorens good catches! Let's tackle those separately, please. ;)

Reply:
> [...] but for the purposes of this PR we might want to address why the Users list is blank and is not showing the message (with or without my commit).

👍 I'll look into it.

Reply:
@msorens can't reproduce. I've created a policy giving my user ALLOW on `*`, and DENY on `iam:users:create`, and see this:
![image](https://user-images.githubusercontent.com/870638/83017733-74f98d80-a024-11ea-93ba-8185b78ff938.png)

Reply:
@sr You exercised a different test case than the one I specified:

> when logged in as a user having no permissions then navigating directly to each page

(No strong preference whether that gets fixed in this PR, though.)